ydb/public/lib/ydb_cli/commands/ydb_dynamic_config.cpp

```diff
@@ -22,6 +22,16 @@ TString WrapYaml(const TString& yaml) {
     return out.Str();
 }
 
+TString WrapStaticConfig(const TString& yaml) {
+    auto doc = NFyaml::TDocument::Parse(yaml);
+    doc.Root().Map().at("config").Insert(doc.Root());
```

Comment:
I'm not sure that it is good practice, but if you ran it under asan and it said ok, then ok